### DISCUSSION TITLE
Config location after installing with one line comand

### DISCUSSION THREAD

**Discussion by jholgate93** [inventree/InvenTree#3967] [Q&A] [answered]
Hi,

I've installed Inventree using the command detailed in https://github.com/inventree/InvenTree/issues/3933 which was successful on Ubuntu 20.04LTS. However, after the install I didn't make a note of the location of the config files. I'd like to customise the logo. Is there a command to get InvenTree to print the location of the config files or is there a default location?

Thanks

**Comment by matmair** [comment, marked as answer]
@jholgate93 you can check the settings for your install - including the location of your config - by running
```
inventree config
```
the default location (following the Linux convention for this type of filed) is `/etc/inventree/config.yml`
